sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/enums/LightNameTests.java

```diff
@@ -0,0 +1,261 @@
+package com.smartdevicelink.test.rpc.enums;
+
+import com.smartdevicelink.proxy.rpc.enums.LightName;
+
+import junit.framework.TestCase;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+/**
+ * This is a unit test class for the SmartDeviceLink library project class :
+ * {@link com.smartdevicelink.rpc.enums.LightName}
+ */
+public class LightNameTests extends TestCase {
+
+	/**
+	 * Verifies that the enum values are not null upon valid assignment.
+	 */
+	public void testValidEnums() {
+		String example = "FRONT_LEFT_HIGH_BEAM";
+		LightName enumFrontLeftHighBeam = LightName.valueForString(example);
+		example = "FRONT_RIGHT_HIGH_BEAM";
+		LightName enumFrontRightHighBeam = LightName.valueForString(example);
+		example = "FRONT_LEFT_LOW_BEAM";
+		LightName enumFrontLeftLowBeam = LightName.valueForString(example);
+		example = "FRONT_RIGHT_LOW_BEAM";
+		LightName enumFrontRightLowBeam = LightName.valueForString(example);
+		example = "FRONT_LEFT_PARKING_LIGHT";
+		LightName enumFrontLeftParkingLight = LightName.valueForString(example);
+		example = "FRONT_RIGHT_PARKING_LIGHT";
+		LightName enumFrontRightParkingLight = LightName.valueForString(example);
+		example = "FRONT_LEFT_FOG_LIGHT";
+		LightName enumFrontLeftFogLight = LightName.valueForString(example);
+		example = "FRONT_RIGHT_FOG_LIGHT";
+		LightName enumFrontRightFogLight = LightName.valueForString(example);
+		example = "FRONT_LEFT_DAYTIME_RUNNING_LIGHT";
+		LightName enumFrontLeftDaytimeRunningLight = LightName.valueForString(example);
+		example = "FRONT_RIGHT_DAYTIME_RUNNING_LIGHT";
+		LightName enumFrontRightDaytimeRunningLight = LightName.valueForString(example);
+		example = "FRONT_LEFT_TURN_LIGHT";
+		LightName enumFrontLeftTurnLight = LightName.valueForString(example);
+		example = "FRONT_RIGHT_TURN_LIGHT";
+		LightName enumFrontRightTurnLight = LightName.valueForString(example);
+		example = "REAR_LEFT_FOG_LIGHT";
+		LightName enumRearLeftFogLight = LightName.valueForString(example);
+		example = "REAR_RIGHT_FOG_LIGHT";
+		LightName enumRearRightFogLight = LightName.valueForString(example);
+		example = "REAR_LEFT_TAIL_LIGHT";
+		LightName enumRearLeftTailLight = LightName.valueForString(example);
+		example = "REAR_RIGHT_TAIL_LIGHT";
+		LightName enumRearRightTailLight = LightName.valueForString(example);
+		example = "REAR_LEFT_BREAK_LIGHT";
```

Comment:
These now need to be updated per a proposal update to `REAR_LEFT_BRAKE_LIGHT` and `REAR_RIGHT_BRAKE_LIGHT`